packages/tsconfig-reference/scripts/tsconfigRules.ts

```diff
@@ -87,7 +87,7 @@ export const recommended = [
   "exactOptionalPropertyTypes",
 ];
 
-type RootProperties = "files" | "extends" | "include" | "exclude";
+type RootProperties = "files" | "extends" | "include" | "exclude" | "reference";
```

Comment:
`references`?

Reply:
Yeah, see CI failure

Reply:
Oops! I made a typo just before committing it 😳